zerver/views/invite.py

```diff
@@ -50,6 +50,9 @@ def invite_users_backend(request: HttpRequest, user_profile: UserProfile,
         streams.append(stream)
 
     do_invite_users(user_profile, invitee_emails, streams, invite_as_admin)
+    recipient_for_event = {user.id for user in user_profile.realm.get_admin_users()}
+    recipient_for_event.add(user_profile.id)
+    invites_changed_event(user_profile, recipient_for_event)
```

Comment:
The events should be sent as a part of do_invite_users and do_revoke_user_invite, to follow the pattern of the other functions in actions.py.
I think you should be able to get the realm from the prereg_user.